components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -604,20 +604,23 @@ public ApiTopUsersListDTO getTopApiUsers(String apiName, String version, String
                         "from " + tableName + " on " + idListQuery.toString() + " within " + getTimestamp(startDate)
                                 + "L, " + getTimestamp(endDate) + "L per '" + granularity + "' select "
                                 + APIUsageStatisticsClientConstants.API_NAME + ", "
+                                + APIUsageStatisticsClientConstants.API_VERSION + ", "
                                 + APIUsageStatisticsClientConstants.API_CREATOR + ", "
                                 + APIUsageStatisticsClientConstants.API_METHOD + ", "
                                 + APIUsageStatisticsClientConstants.APPLICATION_ID + ", "
                                 + APIUsageStatisticsClientConstants.API_RESOURCE_TEMPLATE + ", "
                                 + APIUsageStatisticsClientConstants.TOTAL_REQUEST_COUNT + " group by "
```

Comment:
```suggestion
                                + sum(" + APIUsageStatisticsClientConstants.TOTAL_REQUEST_COUNT
                            + ") as total_request_count,
```

Else only one of the values will be given

Reply:
Modified as suggested.